mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/uwazi/entry/UwaziEntryFragment.kt

```diff
@@ -163,6 +200,10 @@ class UwaziEntryFragment :
         vaultFile.let { uwaziFormView.setBinaryData(it) }
     }
 
+    private fun putRelationShipEntitiesInForm(entitiesList: String) {
+        entitiesList.let { uwaziFormView.setBinaryData(it) }
+    }
```

Comment:
any reason why we are using let scope function here ? unless `entitiesList` is nullable?